### ISSUE TITLE
How to use it?

### ISSUE THREAD

**Issue by micdev42** [oLucasRez/greek-alphabet-snippets#1]
Hi, I have a .md (text) file open in vscode. How do I insert a lowercase lambda? 
Thanks

**Comment by nepomnyi** [comment]
I totally agree with @micdev42 ! The explanation in the README file is insufficient. It should be more detailed. Still, after several trials and errors, I have managed to type a Greek letter. I'm using the latest VScode with Python plugin on the latest Windows11. Put your cursor in a free space of VScode. In all capitals, type DELTA and hit Enter. Your "DELTA" gets converted to the corresponding Greek letter. To get the small Greek delta, type delta and hit Enter.

**Comment by micdev42** [comment]
> In all capitals, type DELTA and hit Enter. Your "DELTA" gets converted to the corresponding Greek letter. To get the small Greek delta, type delta and hit Enter.

Great. Thanks for the help. Works for me!
